frontend/src/pages/projects/pvc/DeletePVCModal.tsx

```diff
@@ -25,82 +26,54 @@ const DeletePVCModal: React.FC<DeletePVCModalProps> = ({ pvcToDelete, onClose })
     setError(undefined);
   };
 
+  const displayName = pvcToDelete ? getPvcDisplayName(pvcToDelete) : 'this storage';
+
   return (
-    <Modal
-      title="Confirm storage delete"
-      variant="small"
+    <DeleteModal
+      title="Delete Storage?"
```

Comment:
All blocks of text should be in sentence case -- capital start, everything else is lowercase.

Also, I am not sure we need to really ask a question with a modal. Although I do not know how UX feels about this.

Reply:
I have no design for this so I referred to the OpenShift console delete project modal, feel free to change it.